### PR TITLE
feat(mdjs): support front matter data

### PR DESCRIPTION
**Pull request by muryoh**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-function parseMatter(s: string): any {
-  if (s.trim().startsWith("{")) {
+function parseMatter(s: string) {
+  if (s.trim().startsWith('{')) {
     // JSON
     return JSON.parse(s);
   }
-  // TODO support YAML
-  return {};
+  const yaml = require('js-yaml');
```
Comment (gqio) [inline src/index.ts]: a really huge dep watch out

Comment (muryoh) [reply src/index.ts]: I'll check for a light one

Comment (gqio) [reply src/index.ts]: there was none at this time that why I didn’t want to use it

Comment (muryoh) [reply src/index.ts]: Lightest I found:

Comment (gqio) [reply src/index.ts]: maybe but looked like unstable to me

Comment (gqio) [reply src/index.ts]: 1M weekly why not

**Pull request merged**